R/friendlyConstructorFunctions.R

```diff
@@ -149,3 +149,84 @@ hasSubstring <- function(string, substring) {
 
   return(v)
 }
+
+#' @export
+rowMean <- function(...) {
+  apply(data.frame(...), 1, mean, na.rm=FALSE)
+}
+
+#' @export
+rowMeanNaRm <- function(...) {
+  apply(data.frame(...), 1, mean, na.rm=TRUE)
+}
+
+#' @export
+rowSum <- function(...) {
+  apply(data.frame(...), 1, sum, na.rm=FALSE)
+}
+
+#' @export
+rowSumNaRm <- function(...) {
+  apply(data.frame(...), 1, sum, na.rm=TRUE)
+}
+
+#' @export
+rowSD <- function(...) {
+  apply(data.frame(...), 1, sd, na.rm=FALSE)
```

Comment:
```suggestion
  apply(data.frame(...), 1, stats::sd, na.rm=FALSE)
```
the same goes for everything until `min`.